### DISCUSSION TITLE
i have gotten github copilot but it doesnt fully work

### DISCUSSION THREAD

**Discussion by justlaughs** [community/community#11232] [Copilot Conversations]
it tired i got the button which has its logo and it doesn't suggest me any code in any of the languages still i can use the button it does things like explaining
![image](https://user-images.githubusercontent.com/98098641/153182445-46d61738-5f9c-4a80-83fb-f48efe36b27f.png)
![image](https://user-images.githubusercontent.com/98098641/153182495-cbf5d051-89ef-4d78-8511-a04c627d0474.png)

**Comment by davecheney** [comment]
I’m sorry you’re having difficulties. I believe copilot is working, perhaps you could try some other examples, or try copilot on a codebase that you’re working on.

**Comment by KTibow** [comment]
You need to make a new line in order for Copilot to make suggestions.
